src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperationTest.java

```diff
@@ -830,6 +863,484 @@ public void failurePropagated() {
       // then result
       assertThat(failure).isEqualTo(exception);
     }
+
```

Comment:
Would it make sense to separate these tests as something like `FindOperationWithSortTest`?

Reply:
Will Do